### PR TITLE
Add more upgrade notes

### PR DESCRIPTION
**Pull request by bernd**
This is the content from https://github.com/Graylog2/graylog2-server/blob/3.2/UPGRADING.rst

### CODE REVIEW

**Review by mpfz0r — DISMISSED** [review]
LGTM, someone else should double check the Dashboard parts

File: pages/upgrade/graylog-3.2.rst [hunk 1]
```diff
@@ -9,6 +9,64 @@ Upgrading to Graylog 3.2.x
    :backlinks: top
 
 
+.. note:: Graylog 3.2.0 comes with a number of migrations that change parts of your existing data fundamentally. Please make sure to have a recent backup of your MongoDB that you can rollback to, before attempting to upgrade from an earlier version.
```
Comment (jalogisch) [inline pages/upgrade/graylog-3.2.rst]: I would make a warn and not only a note

**Review by jalogisch — DISMISSED** [review]
👍 

Only a single item I would change. But that is not fundamental at all.

**Comment by bernd** [comment]
> LGTM, someone else should double check the Dashboard parts

The dashboards parts have been written by the search team, so I guess they are fine. :smiley:

File: pages/upgrade/graylog-3.2.rst (hunk 1 above)
Comment (bernd) [reply pages/upgrade/graylog-3.2.rst]: I updated the PR to make this a warning.

**Review by mpfz0r — APPROVED** [review]

**Pull request merged**